Dockerfile

```
@@ -7,7 +7,9 @@ WORKDIR /app
 COPY pyproject.toml poetry.lock ./
 COPY src ./src
 
-RUN pip install poetry && \
+RUN apt-get update && \
+    apt-get install -y ffmpeg && \
+    pip install poetry && \
```

```suggestion
    apt-get install -y ffmpeg
RUN pip install poetry && \
```

I'd do separate run layers for apt and pip/poetry so it can cache when poetry breaks (way more likely than apt breaking)